docs/hub/security.md

```diff
@@ -4,6 +4,54 @@ title: Security and trust on the Hugging Face Hub
 
 # Security and trust on the Hugging Face Hub
 
+## User Access Tokens
+
+### What are User Access Tokens?
+
+User Access Tokens are the preferred way to authenticate an application or a notebook to the HuggingFace services. You can manage your access tokens in your [settings](https://huggingface.co/settings/tokens).
+
+![/docs/assets/hub/access-tokens.png](/docs/assets/hub/access-tokens.png)
+
+Access tokens allow applications and notebooks to impersonate you, with restricted permissions. Each token is given a role, among the following:
+
+- `read`: tokens with this role can only be used to provide read access to repositories you could read. That includes public repositories and private repositories that you or an org you're a member of own. Use this role if you only need to read content from the HuggingFace hub (eg, when doing inference).
+
+- `write`: tokens with this role additionally grant write access to the repositories you have write access to. Use this token if you need to push content to a repository (eg, when training a model).
+
```

Comment:
No doc for the admin token?

Reply:
We are discussing wether or not we want to keep it, as it's essentially useless now - but could grant programmatic access to organizations' settings in the future